tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -429,15 +429,21 @@ Then('file {string} should not be listed on the webUI', function(file) {
   return client.page.FilesPageElement.filesList()
     .isElementListed(file, 'file', client.globals.waitForNegativeConditionTimeout)
     .then(state => {
-      return client.assert.ok(!state, `Error: File ${file} is listed on the filesList`)
+      const message = state
+        ? `Error: File '${file}' is listed on the filesList`
+        : `File '${file}' is not listed on the filesList`
```

Comment:
When `assert.ok` passes, the message still gets displayed. So that is why it is useful to have the appropriate message.